src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java

```diff
@@ -81,59 +87,70 @@ public ReadsPreprocessingPipelineTestData(Class<?> clazz) {
         kvRefBasesiReads = Arrays.asList(
                 KV.of(FakeReferenceSource.bases(spannedReadInterval), Lists.newArrayList(reads.get(1), reads.get(0))),
                 KV.of(FakeReferenceSource.bases(readIntervals.get(2)), Lists.newArrayList(reads.get(2))),
-                KV.of(FakeReferenceSource.bases(readIntervals.get(3)), Lists.newArrayList(reads.get(3))));
+                KV.of(FakeReferenceSource.bases(readIntervals.get(3)), Lists.newArrayList(reads.get(3))),
+                KV.of(FakeReferenceSource.bases(readIntervals.get(4)), Lists.newArrayList(reads.get(4)))
+        );
 
         kvReadsRefBases = Arrays.asList(
-                KV.of(reads.get(0), getBases(reads.get(0).getStart(), reads.get(0).getEnd())),
-                KV.of(reads.get(1), getBases(reads.get(1).getStart(), reads.get(1).getEnd())),
-                KV.of(reads.get(2), getBases(reads.get(2).getStart(), reads.get(2).getEnd())),
-                KV.of(reads.get(3), getBases(reads.get(3).getStart(), reads.get(3).getEnd()))
+                KV.of(reads.get(0), getBases("1", reads.get(0).getStart(), reads.get(0).getEnd())),
+                KV.of(reads.get(1), getBases("1", reads.get(1).getStart(), reads.get(1).getEnd())),
+                KV.of(reads.get(2), getBases("1", reads.get(2).getStart(), reads.get(2).getEnd())),
+                KV.of(reads.get(3), getBases("1", reads.get(3).getStart(), reads.get(3).getEnd())),
+                KV.of(reads.get(4), getBases("2", reads.get(4).getStart(), reads.get(4).getEnd()))
         );
 
         variants = Lists.newArrayList(
                 new SkeletonVariant(new SimpleInterval("1", 170, 180), true, false, new UUID(1001, 1001)),
```

Comment:
Verified, they are very brittle.  Changing `REFERENCE_SHARD_SIZE` to 99999 starts causes test failures.